lib/init.coffee

```diff
@@ -211,10 +211,18 @@ module.exports =
           messages = @parseSchematronMessages(textEditor, output)
         else
           messages = @parseSchemaMessages(textEditor, output)
-        for message in messages
-          message.type = 'Error'
-          message.text = message.text + ' (' + schemaUrl + ')'
-          message.filePath = textEditor.getPath()
+        if messages.length
+          for message in messages
+            message.type = 'Error'
+            message.text = message.text + ' (' + schemaUrl + ')'
+            message.filePath = textEditor.getPath()
+        else if output.indexOf('- validates') is -1
+          messages.push({
+            type: 'Error'
+            text: output
+            filePath: textEditor.getPath()
+            range: [[0, 0], [0, 0]]
```

Comment:
That's why I always keep my own PRs open for a day or two. You always have a good comment :wink: Thanks - updated in c9e5bf81504543deb4196194999301ece6f45673